content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,276 @@ ifndef::include-only-once[]
+
+Measurement::
+(To be defined)
+
+* Source: 
+* Notes: We would take different meanings compared to an automotive testing domain.
+
+Model::
+(To be defined)
+
+* Source: 
+* Notes: Differentiate between 3D model and other models, e.g. sensor models, logics models, vehicle models.
+
+Multispectral/hyperspectral::
+Wide range of wavelengths.
+
+* Source: 
+* Notes:
+
+Normals (face, vertex normals)::
+(To be defined)
+
+* Source: 
+* Notes: Look up normal vector on surface.
+
+Object::
+An object ID can be assigned, discrete meshes, defined 3D boundaries.
+
+* Source: 
+* Notes:
+
+Object Part / Subtype::
+(To be defined)
+
+* Source: 
+* Notes: What is an object and what is an object part?
+
+Object Type::
+Class of objects with common properties
+
+* Source: 
+* Notes:
+
+Open loop::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Opacity | Translucence | Transparency::
+(To be defined)
+
+* Source: 
+* Notes: Taken over from terminology collection list on Sharepoint.
+
+Parameter File::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Path tracing::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Perception::
+"Machine perception is to reliably detect all the other road users relevant to the operation of the automated driving, and to assign them correctly to the traffic infrastructure."
+
+* Source: Klaus Dietmayer, Predicting of Machine Perception for Automated Driving, 2016
+* Notes:
+
+Physical (as in physical parameter)::
+A physical property is any property of a physical system that is measurable. 
+
+* Source: Mark, Burgin (2016-10-27). Theory Of Knowledge: Structures And Processes
+* Notes:
+
+Post (Rendering) Effects::
+Rendering techniques to improve fidelity while maintaining performance, that are typically non-physical (or in other words fake)
+
+* Source: 
+* Notes:
+
+Radar sensor::
+(To be defined)
+
+* Source:
+* Notes:
+
+Ray tracing::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Real time / offline::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Roughness::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Scene::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Scenario.::
+A description of the behavior or temporal evolution of physical objects and environmental conditions on the driving infrastructure over an interval of time, including the movement of traffic participants or the change of environmental conditions.
+
+* Source: 
+* Notes: OpenSCENARIO standard, 2024
+
+Signal (Energy) Propagation::
+"During signal [or energy] propagation, the signal travels through the environment and possibly interacts with objects and particles."
+
+* Source: Linnhoff et al., Towards Serious Perception Sensor..., 2021
+* Notes:
+
+Simulation environment::
+(To be defined)
+
+* Source: 
+* Notes:
+
+Simulation Platform::
+Difference between platform (possibly handling rendering, physics etc.)  and an orchestrator (only handling model execution and interfacing)
+
+* Source: 
+* Notes:
+
+Simulation::
+(Look up definition)
+
+* Source: 
+* Notes:
+
+Texture::
```

Comment:
Maybe this term is to broad and does not need to be defined specifically for OpenMATERIAL?